user-guide/Advanced_Functionality/Swarming/Swarming.md

```diff
@@ -4,25 +4,80 @@ uid: Swarming
+
+- Swarming is not available in DataMiner Systems with a [storage per DMA setup](xref:Configuring_storage_per_DMA) (Cassandra or MySQL).
+
+- Swarming is not available in DataMiner Systems with [Failover](xref:About_DMA_Failover).
+
+- Swarming is not available in DataMiner Systems where the [*LegacyReportsAndDashboards* soft-launch option](xref:Overview_of_Soft_Launch_Options#legacyreportsanddashboards) is enabled.
+
+- Swarming is currently limited to [basic elements](xref:SwarmingElements). For [bookings](xref:SwarmingBookings), swarming is supported if the *BookingSwarming* [soft-launch option](xref:SoftLaunchOptions) is enabled.
+
+> [!NOTE]
+> [Prerequisite checks](xref:EnableSwarming#running-a-prerequisites-check) are in place to prevent the enabling of the Swarming feature when non-supported objects are present. Where possible, you will also be prevented from configuring or creating these on a Swarming-enabled system.
+
+Below you can find a complete overview of the differences between a system using DataMiner 10.5.1 or higher with or without Swarming.
+
+|                                                  | 10.5.1+&nbsp; systems without Swarming | 10.5.1+ systems with Swarming  |
+|--------------------------------------------------|----------------------------------------|--------------------------------|
+| Element configuration                            | On disk (element.xml)                  | In database                    |
+| Alarm IDs                                        | Per DataMiner Agent                    | Per element                    |
+| Database per DMA                                 | Supported                              | Not supported                  |
+| Scripts & QActions using legacy alarm references | Supported                              | Not supported                  |
+| Legacy Reports & Dashboards                      | Supported                              | Not supported                  |
+| Failover                                         | Supported                              | Not supported                  |
+| Offload database                                 | Supported                              | Not supported*                 |
+| Contributing bookings                            | Supported                              | Not supported*                 |
+| SLA elements                                     | Supported                              | Supported but not swarmable*   |
+| Enhanced services                                | Supported                              | Supported** but not swarmable* |
+| Spectrum elements                                | Supported                              | Supported but not swarmable*   |
+| Redundancy group elements                        | Supported                              | Supported but not swarmable*   |
+| Elements exporting DVEs of functions             | Supported                              | Supported but not swarmable*   |
```

Comment:
Should this be DVEs *or* functions? If yes, I'd rephrase this as follows to keep the terminology in line with the rest of the documentation:
```suggestion
| DVE child and parent elements and virtual function resources | Supported                              | Supported but not swarmable*   |
```

Reply:
Would this be OK? The original line does not mention that the exported DVEs/functions also can't be swarmed, but I assume this is also the case?
```suggestion
| DVE and virtual function child and parent elements            | Supported                              | Supported but not swarmable*   |
```